src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import TodoForm from './TodoForm';
```

Comment:
If you don't need state, you don't need your component to be a class, you can turn it into a function... if ya want to.